devnet/cmd/scale-down-ecs-service/service_scaler_test.go

```diff
@@ -0,0 +1,498 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/devnet/cmd/scale-down-ecs-service/mocks"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/aws/aws-sdk-go/service/ecs/ecsiface"
+	"github.com/stretchr/testify/mock"
+)
+
+//go:generate mockery --srcpkg=github.com/aws/aws-sdk-go/service/ecs/ecsiface --name ECSAPI --case underscore
```

Comment:
Maybe write it to `ecsapi_mock_test.go` locally in this package so it's not exposed in the Go API, especially if it's only used in this test file

Reply:
🤔  but this creates it in a `mocks` folder within the `scale-down-ecs-service` folder.

Reply:
I wasn't able to get it to generate inside the same package.  The `--inpackage` flag creates the mock inside the package that the interfaces lives in.  I renamed it to append "_test.go".

Reply:
nm, scratch that.  Since the mocks folder doesn't include any non-test files, it complains.  Going to leave it as is.  At some point, I'm going to move the whole devnet folder to another repo, so that will clean things up nicely.

Reply:
Ah ok sorry I forgot how to do it with `mockery` so I couldn't suggest some alternative code, maybe it's not possible.
Can you name the mocked interface to something else than the original interface name with mockery?